textproc/ocrmypdf/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                ocrmypdf
 version             16.1.2
-revision            0
+revision            1
```

Comment:
this doesn't change the default, which remains 3.12 so no need to increase the `revision`

Reply:
got it